Readme.md

```diff
@@ -18,7 +31,7 @@ Start all the basic services
 
 **Note:** Depending on how you installed docker, you might have to run the docker commands through `sudo`
 ```
-docker-compose up --build -d
+docker-compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.prod.yml up --build -d
```

Comment:
The readme is meant to only contain instructions for development (and we can link to the runbook for prod). This is so that new developers find it easy to get started.

Reply:
Ah, then I will put it up as a note at the top of the readme.